version1/04 - Web services.md

```diff
@@ -1267,13 +1283,13 @@ Content-Type: application/json
   - only return access log entries ending at this datetime (inclusive)
 - **userName** (optional)
   - the username to filter by
-- **objectType** (conditionally optional)
+- **objectType**
   - the object type to filter by \[Saksmappe, Mappe, Moetemappe, Journalpost, Basisregistrering, Moeteregistrering,
     Arkivnotat, Dokument, Dokumentversjon\]
-  - must be provided if objectUUIDs is
-- **objectUUIDs** (conditionally optional)
+  - optional, but recommended when objectUUIDs is specified
+- **objectUUIDs**
   - the list of object system ids to filter by
-  - must be provided if objectType is
+  - optional
```

Comment:
I don't think the `objectUUIDs` is optional.

Reply:
For access logs, UUIDs are optional. We only require an ID for the changelog.

Reply:
Hmm, yes, my bad :sweat_smile: